PRChecker/Shared/GraphQL/NetworkService.swift

```diff
@@ -77,25 +86,28 @@ final class NetworkSerivce {
         self.useLegacyQuery = useLegacyQuery
     }
     
-    func getAllPRs() -> AnyPublisher<(String, [AbstractPullRequest]), Error> {
+    func getAllPRs() -> AnyPublisher<NetworkPRResult, Error> {
         getAllPRs(for: username)
     }
     
-    func getAllPRs(for username: String) -> AnyPublisher<(String, [AbstractPullRequest]), Error> {
-        let assignedQuery = "is:pr assignee:\(username) archived:false"
-        let requestedQuery = "is:pr review-requested:\(username) archived:false"
+    func getAllPRs(for username: String) -> AnyPublisher<NetworkPRResult, Error> {
+        let assignedQuery = "is:pr assignee:\(username) archived:false sort:updated"
+        let requestedQuery = "is:pr review-requested:\(username) archived:false sort:updated"
+        let reviewedQuery = "is:pr reviewed-by:\(username) archived:false sort:updated"
         
-        return Publishers.Zip(getPR(with: assignedQuery), getPR(with: requestedQuery))
+        return Publishers.Zip3(getPR(with: assignedQuery), getPR(with: requestedQuery), getPR(with: reviewedQuery))
```

Comment:
The count and variety of PRs has become more again lol
(Totally unrelated to this PR) I'm thinking about that maybe we should only show `Requested PR` in menu bar..since it actually acts like a notification bar?

Reply:
Yeah that would be a good idea since we don't need the already reviewed items.